icon/iiss/prepmanager.go

```diff
@@ -377,17 +361,10 @@ func (pm *PRepManager) ChangeBond(oBonds, nBonds icstate.Bonds) error {
 
 		if value.Sign() != 0 {
 			ps := pm.state.GetPRepStatus(owner)
-			if ps == nil {
-				// Someone tries to bond to a PRep which has not been registered
-				panic(errors.Errorf("Failed to set bonded value to PRepStatus"))
-			} else {
-				newPs = ps.Clone()
-			}
-			if !newPs.IsActive() {
+			if !ps.IsActive() {
```

Comment:
It there is no PRepStatus for owner, GetPRepStatus() return new ps.

New ps is not increase bondedToInactiveNode because default value of ps.status is icstate.Active.

Need to fix PRepStatus constructor, NewPRepStatus(), to make instance has icstate.NotReady Status.

Reply:
If fix NewPRepStatus(), ChangeDelegation() should work as well.

Reply:
```
func NewPRepStatus(owner module.Address) *PRepStatus {
	return &PRepStatus{
		owner:      owner,
		grade:      Candidate,
		delegated:  new(big.Int),
		bonded:     new(big.Int),
		vFail:      0,
		vTotal:     0,
		lastState:  None,
		lastHeight: 0,
		status:     NotReady,
	}
}
```
I added NotReady status in the constructor of PRepStatus.
Is is okay?
much appreciated